lib/common.js

```diff
@@ -94,7 +124,17 @@ const overrideRequests = function(newRequest) {
       get: overriddenGet,
     }
 
-    module.request = function(options, callback) {
+    module.request = function(input, options, callback) {
```

Comment:
So we don't end up with three copies of this function, could you extract it to a helper function in `common.js`?